### PR TITLE
productive-seas-fish-and-shellfish-stocks-fished-sustainably

### PR DESCRIPTION
**Pull request by bailed22**
Transformation carried out with 
1. wrangling.py
2. config.json
3. output.csv
4. concatenated source data

### CODE REVIEW

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Productive seas fish and shellfish stocks fished sustainably",
+    "publisher": "https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs",
+    "creator": "https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs",
+    "summary": "The proportion of stocks fished sustainably in both the long term and in recent years",
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:6]: Replace with:
"Proportion of commercial fish and shellfish stocks that are within safe biological limits and fished sustainably"

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "fished",
+        "commercial-fish",
+        "shellfish-stocks",
+        "fished-sustainably",
+        "marine-fish",
+        "united-kingdom"
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:15]: Delete "united-kingdom"

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "fished",
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:10]: replace with:
"fish"

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "columns": {
+        "Year": {
+            "type": "dimension",
+            "from_template": "year"
+        },
+        "Category": {
+            "type": "dimension"
+        },
+        "Measure": {
+            "type": "measures"
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:30]: You need to list the two measures. Take a look at Woodland area local authority.
For example:

```
"Measure":{
            "type": "measures",
            "values" : [
                {
                   "label": "Standard Area", 
                   "from_existing": "https://purl.org/csv-cubed/resources/measures#land-area"
                },
                {
                    "label": "Woodland Area",
                    "from_existing": "https://purl.org/csv-cubed/resources/measures#land-area"
                },
                {
                    "label": "Woodland area as a portion of standard area",
                    "from_existing": "https://purl.org/csv-cubed/resources/measures#portion"
                }
            ]
  },
```

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+@click.command()
+@click.argument("input", type=click.Path(exists=True, path_type=Path))
+@click.option("--output", default=Path("./output.csv"), type=click.Path(path_type=Path))
+def wrangle(input: Path(), output: Path()) -> None:
+    
+    df = pd.read_csv(input, na_values="na")
+
+    df['Value'] = df['Value'].astype(float).round(2)
+
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py:14]: You can delete the empty lines 12 and 14.
It is a short code; no need for the spaces.

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:2]: check the latest config files on `family-ess` repo to see if the schema has been updated.

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+from email.policy import default
+import click
+import pandas as pd
+from pathlib import Path
+
+@click.command()
+@click.argument("input", type=click.Path(exists=True, path_type=Path))
+@click.option("--output", default=Path("./output.csv"), type=click.Path(path_type=Path))
+def wrangle(input: Path(), output: Path()) -> None:
+
```
Comment (osamede20) [inline productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py:10]: It is also good to arrange the dataframe columns in the order in which they should appear

**Review by osamede20 — CHANGES_REQUESTED** [review]
Please check if there are other things besides the requested changes

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json (hunk 6 above)
Comment (bailed22) [reply productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:2]: No updated version availale

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json (hunk 1 above)
Comment (bailed22) [reply productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:6]: adjusted as requested

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json (hunk 2 above)
Comment (bailed22) [reply productive-seas-fish-and-shellfish-stocks-fished-sustainably/productive-seas-fish-and-shellfish-stocks-fished-sustainably.json:15]: Done

File: productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py (hunk 5 above)
Comment (bailed22) [reply productive-seas-fish-and-shellfish-stocks-fished-sustainably/wrangling.py:14]: Done

**Pull request merged**